### PR TITLE
Biggest winner paginated

### PR DESCRIPTION
**Pull request by BernardHan**

### CODE REVIEW

File: src/graphql/queries/biggest-winners.js [hunk 1]
```diff
@@ -64,7 +64,28 @@ module.exports = async (
     }));
   });
   await Promise.all(promises);
-  winnings = _orderBy(winnings, ['amount'], ['desc']);
+  let items = winnings;
```
Comment (dwalintukan) [inline src/graphql/queries/biggest-winners.js]: dont need to reassign, just do `{ items: winnings }` at the return

File: src/graphql/queries/biggest-winners.js [hunk 2]
```diff
@@ -64,7 +64,28 @@ module.exports = async (
     }));
   });
   await Promise.all(promises);
-  winnings = _orderBy(winnings, ['amount'], ['desc']);
```
Comment (dwalintukan) [inline src/graphql/queries/biggest-winners.js]: why remove the ordering? youll have to implement `orderBy` param for the query then.

Comment (BernardHan) [reply src/graphql/queries/biggest-winners.js]: the orderBy gives weird order, one result was 7, 200, 0, don't know how.

So I'm just using array.sort.

Comment (dwalintukan) [reply src/graphql/queries/biggest-winners.js]: hm oh prob because its sorting by Strings instead of numbers

**Comment by dwalintukan** [comment]
🚢

**Pull request merged**